api/src/zero/service/postgres.clj

```diff
@@ -70,7 +70,7 @@
   [tx {:keys [cromwell id items] :as _workload}]
   (try
     (let [workflows (get-table tx items)]
-      (run! (partial update-workflow-status! tx cromwell items) workflows)
+      (pmap (partial update-workflow-status! tx cromwell items) workflows)
```

Comment:
I'm not sure about this:
- whether we should do this?
- whether this makes sense?

Reply:
Is there any performance advantage to this?
I'd suggest not making threading changes for a hot-fix.